root/static/scripts/edit/components/ExternalLinkAttributeDialog.js

```diff
@@ -215,7 +242,7 @@ const ExternalLinkAttributeDialog = (props: PropsT): React.MixedElement => {
                 className="positive"
                 disabled={hasErrors}
                 onClick={() => handleConfirm(closeAndReturnFocus)}
-                type="submit"
+                type="button"
```

Comment:
Is this change needed to resolve something?  We have an `onSubmit` handler and I don't believe that it would be triggered with this anymore (so pressing enter to submit the dialog is likely broken by this).

Reply:
IIRC this was meant to avoid the "Form submission canceled because the form is not connected" console error.

Reply:
Yeah, I tested this, and you're right it breaks pressing enter. I changed it back, and it works, but prints "Form submission canceled because the form is not connected" to the console every time, it seems. Still better than breaking the dialog submission with Enter, but ideally we can figure a way to avoid that.

Reply:
I pushed a commit which seems to fix that (I guess it was the `onClick` handler).